src/redux/paymentMethodsSlice.js

```diff
@@ -0,0 +1,64 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { getPaymentMethods, submitPaymentWithoutDetails } from 'api';
+
+const initialState = {
+  paymentMethods: null,
+  loading: false,
+  error: null,
+  availablePaymentMethods: null
+};
+
+export const fetchPaymentMethods = createAsyncThunk(
+  'paymentMethods/fetchPaymentMethods',
+  async () => {
+    const { paymentResponse } = await getPaymentMethods();
+    return paymentResponse;
+  }
```

Comment:
Get payment methods (https://developers.cleeng.com/reference/fetch-payment-method-ids) returns a `paymentMethods` response, so should be: 
` const { paymentMethods } = await getPaymentMethods();`